libs/scss-foundation/docs/modules/sizes.md

```diff
@@ -0,0 +1,17 @@
+# Sizes methods
+
+Sizes utilities for setting dimensions for with and height.
+
+## Items
+
+@use scss-foundation/src/modules/sizes/set-sizes;
+
+```
+@include set-sizes.box-dimensions(height, 200px);
+```
+
+Set height property and it's value
```

Comment:
line 3, typo wiDth

line 13, its value

I guess this documentation file hasn't gone through spell checker/chat GPT as we should always do.